api/src/transactions.rs

```diff
@@ -341,7 +346,62 @@ impl TransactionsApi {
             self.context.max_transactions_page_size(),
         );
         let api = self.clone();
-        api_spawn_blocking(move || api.list_by_account(&accept_type, page, address.0)).await
+        api_spawn_blocking(move || api.list_ordered_txns_by_account(&accept_type, page, address.0))
+            .await
+    }
+
+    /// Get account transaction summaries
```

Comment:
Can you comment on how the range works, e.g. if the end of the range is included or not.

Reply:
Done.